visidata/loaders/toml.py

```diff
@@ -0,0 +1,51 @@
+from visidata import (
```

Comment:
```suggestion
from visidata import ColumnItem, PythonSheet, VisiData, asyncthread, deduceType, vd
```

Reply:
[Let's not nitpick formatting, esp outside of visidata core.]